File: week_01/code_wars/shhong/200825_Multiples_of_3or5.py
def solution(number):
    total = sum(num * 3 for num in range(1, number//3 + 1) if (num * 3) < number)
    total += sum(num * 5 for num in range(1, number//5 + 1) if (num * 5) < number)
    total -= sum(num * 15 for num in range(1, number//15 + 1) if (num * 15) < number)
    return total

File: week_01/code_wars/chSung/200825_multiples_of_3_or_5.java
public class Solution {

    public int solution(int number) {
        int sum = 0;

        for (int i = 1; i * 3 < number; i++) {
            sum += i * 3;
        }
        for (int i = 1; i * 5 < number; i++) {
            if ((i * 5) % 15 == 0) {
                continue;
            }
            sum += i * 5;
        }

        return sum;
    }
}
